src/experiment/launch_experiment.jl

```diff
@@ -66,14 +68,14 @@ function launch_experiment!(
             dt = @elapsed search!(model, strategy, variableHeuristic, valueSelectionArray[j], out_solver=out_solver)
 
             if isa(valueSelectionArray[j], LearnedHeuristic)
-                verbose && print(", Visited nodes: ", model.statistics.numberOfNodes)
+                verbose && print(", Visited nodes with learnedHeuristic : ", model.statistics.numberOfNodes)
             else
-                verbose && print(" vs ", model.statistics.numberOfNodes)
+                verbose && println(" vs Visited nodes with basic Heuristic n°$(j-1) : ", model.statistics.numberOfNodes)
             end
-
-            # bestsolutions[i, j] = model.objectiveBound + 1
+            bestsolutions[i, j] = model.objectiveBound + 1
             nodevisited[i, j] = model.statistics.numberOfNodes
 
+            #TODO understand what this line is doing
             if j == 2
                 set_postfix(iter, Delta=string(nodevisited[i, 1] - nodevisited[i, 2]))
```

Comment:
This line is to update the progress bar, to show the current difference between the basic heuristic and the learned one (in terms of number of nodes).